src/control-plane/backend/lambda/api/router/reporting.ts

```diff
@@ -19,13 +19,21 @@ const reporting_project = express.Router();
 const reportingServ: ReportingServ = new ReportingServ();
 
 reporting_project.post(
-  '',
+  '/funnel',
```

Comment:
is it sankey or funnel visual?

Reply:
it funnel visual.  
sankey visual is used for path analysis.

plan to use 
/funnel  
/event
/path
/retention 
for our 4 type exploration dashboards.